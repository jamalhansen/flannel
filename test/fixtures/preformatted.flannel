This is a paragraph

_   This is preformatted text
        it's cool

_   and Spiffy

_ This is preformatted too
_

_
 So is this
_

This is a paragraph