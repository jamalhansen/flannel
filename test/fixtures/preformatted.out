<p>This is a paragraph</p>

<pre>   This is preformatted text
        it's cool

   and Spiffy</pre>

<pre>This is preformatted too</pre>

<pre> So is this</pre>

<p>This is a paragraph</p>